src/main/scala/dsptools/numbers/package.scala

```diff
@@ -0,0 +1,14 @@
+package dsptools
+
+package object numbers extends AllSyntax with AllImpl with spire.syntax.RingSyntax
```

Comment:
I read somewhere that sometimes people will have a lowercase package (I think) without the implicit and an uppercase version of the package (I think) with the implicits. Is that something we should think about doing?